README.md

```diff
@@ -151,3 +151,21 @@ to create a 'Service' and a 'Profile' tied to the developer portal accounts.
 
 You should receive an API-Key (in the developer portal) and an email
 with a fake fiscal code you can use to start testing the Digital Citizenship API.
+
+## ADB2C Sign-in / Sign-up form style
```

Comment:
I don't understand this `ADB2C` acronym, can we use something more familiar? what does it mean?

Reply:
it's 'active directory business to clients (there's a link to the azure docs at the beginning of this readme): https://azure.microsoft.com/it-it/services/active-directory-b2c/'

that's the MS way to call "an active directory where you can login non-microsoft accounts" (ie. with any email)